public/javascripts/script.js

```diff
@@ -1,4889 +0,0 @@
-  document.getElementById("featureEasy").classList.toggle("border-solid");
-  document.getElementById("featureEasy").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-document.getElementById("featureFeature").addEventListener("click", () => {
-  if (!checkedFilters.features) {
-    checkedFilters.features = true;
-  } else {
-    checkedFilters.features = false;
-  }
-  document.getElementById("featureFeature").classList.toggle("bg-blue-100");
-  document.getElementById("featureFeature").classList.toggle("border-blue-600");
-  document.getElementById("featureFeature").classList.toggle("border-solid");
-  document.getElementById("featureFeature").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-document.getElementById("featureSupport").addEventListener("click", () => {
-  if (!checkedFilters.customerSupport) {
-    checkedFilters.customerSupport = true;
-  } else {
-    checkedFilters.customerSupport = false;
-  }
-  document.getElementById("featureSupport").classList.toggle("bg-blue-100");
-  document.getElementById("featureSupport").classList.toggle("border-blue-600");
-  document.getElementById("featureSupport").classList.toggle("border-solid");
-  document.getElementById("featureSupport").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-document.getElementById("featureValue").addEventListener("click", () => {
-  if (!checkedFilters.valueForMoney) {
-    checkedFilters.valueForMoney = true;
-  } else {
-    checkedFilters.valueForMoney = false;
-  }
-  document.getElementById("featureValue").classList.toggle("border-blue-600");
-  document.getElementById("featureValue").classList.toggle("bg-blue-100");
-  document.getElementById("featureValue").classList.toggle("border-solid");
-  document.getElementById("featureValue").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-document.getElementById("featureDistribution").addEventListener("click", () => {
-  if (!checkedFilters.distribution) {
-    checkedFilters.distribution = true;
-  } else {
-    checkedFilters.distribution = false;
-  }
-  document.getElementById("featureDistribution").classList.toggle("bg-blue-100");
-  document.getElementById("featureDistribution").classList.toggle("border-blue-600");
-  document.getElementById("featureDistribution").classList.toggle("border-solid");
-  document.getElementById("featureDistribution").classList.toggle("border-4");
-  runFilters(checkedFilters);
-});
-
-const runFilters = (checkedFilters) => {
-  const companies = Array.from(document.getElementsByClassName("companyDOM"));
-  companies.forEach((company) => {
-    let doesThisCompanyPass = true;
-    for (key in checkedFilters) {
-      if (checkedFilters[key]) {
-        console.log("CHECKING COMPANIES FOR", key);
-        console.log("COMPANY SCORE FOR", key, Number(company.getAttribute(`data-${key}`)));
-        if (Number(company.getAttribute(`data-${key}`)) < 4) {
```

Comment:
Why is this 4? This seems like an abstract number which indicates it could be a bit of hack. Best to consider what you're trying to do here and if it's the correct approach. 

If it is, it's always good to avoid including what are called 'magic numbers' in code. Instead, we want to assign that number to a variable that explains what it's doing (usually we use UPPER_CASE naming for this like below:

```
const MAGIC_NUMBER_NAME_GOES_HERE = 4

Reply:
Then we can use the below:

```
if (Number(company.getAttribute(`data-${key}`)) < MAGIC_NUMBER_NAME_GOES_HERE) {